### PR TITLE
Wait for xterm start before typing commands

### PR DESCRIPTION
**Pull request by kravciak**
Fix for: https://openqa.suse.de/tests/621441#step/smt/5
Local run: http://dhcp91.suse.cz/tests/2867

### CODE REVIEW

File: tests/x11/smt.pm [hunk 1]
```diff
@@ -7,18 +7,19 @@
...
+# Maintainer: Jozef Pupava <jpupava@suse.com>
 
 use base "x11test";
 use strict;
 use warnings;
 use testapi;
 
 sub run() {
-    x11_start_program("xterm -geometry 150x35+5+5");
+    x11_start_program("xterm");
```
Comment (dzedro) [inline tests/x11/smt.pm]: [this](http://dhcp91.suse.cz/tests/2866#step/smt/21) is better readable than in a bigger xterm ?

**Comment by kravciak** [comment]
Fixed

**Comment by okurz** [comment]
typo in commit message and PR description

**Comment by kravciak** [comment]
fixed

File: tests/x11/smt.pm [hunk 2]
```diff
@@ -57,7 +58,7 @@ sub run() {
     }
     assert_script_run 'smt-sync', 600;
     assert_script_run 'smt-repos';
-    type_string "killall xterm\n";
+    send_key "alt-f4";
```
Comment (okurz) [inline tests/x11/smt.pm]: see cc19fd6 regarding this change and [poo#10646](https://progress.opensuse.org/issues/10646). If there is a good reason for using 'alt-f4' instead then the same should be used all over the place where 'killall xterm\n' is also used.

**Comment by kravciak** [comment]
fixed

**Pull request merged**